### ISSUE TITLE
A11y : Cannot interact with Insights panels via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46476]
**Steps to reproduce**:
Inside Explore:

When in `Insights`, a person using keyboard-only navigation interact with the panels

**Actual Result**:

Still visible but degraded experience compared to a mouse user who can hover over the graph to see info details

**Expected Result**

Users without a pointer device should have a way to access details

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by gabor** [comment]
hi @zuchka sorry, could you please explain what you mean by the `Insights` panels in explore?

**Comment by yaelleC** [comment]
Hi @gabor I just re-checked and I believe this issue is now fixed (I don't know when though?). I could not find an `insights` panel but checked with Logs and Graph and both work fine (see below).

| Mouse over | Tab interaction inside Log panel + arrow keys | Graph panel |
| ------------ | ---------------------------------------------| ------ |
| ![Screenshot 2022-07-29 at 09 05 40](https://user-images.githubusercontent.com/42030685/181703002-9594ed30-6776-4d59-9e80-aec0378f0262.png) | ![Screenshot 2022-07-29 at 09 17 39](https://user-images.githubusercontent.com/42030685/181704857-fa5d4861-d7ed-49a9-898d-4de5b0592d2e.png) | ![Screenshot 2022-07-29 at 09 18 55](https://user-images.githubusercontent.com/42030685/181704892-1633b74d-597a-4255-ae56-5dff75443f42.png) |


Looks to me like the experience is no longer degraded for keyboard users 👍 So we could close that issue?

@Elfo404 , you might have more context on this issue and the `insights` panel?

**Comment by gabor** [comment]
thanks for the info @yaelleC , i'll close the issue for now. if @Elfo404 has some extra info we can always reopen it.
